sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -625,6 +625,7 @@ def to_type_hint(self):
 
 
 class _CustomBigQuerySource(BoundedSource):
+
   def __init__(
```

Comment:
Would you mind sharing the motivation with me on this. Its not entirely clear why you are making this change to me.

Can you make equivalent changes in the Java implementation as well? For a consistent experience

Reply:
Yes, I'll make the change on the Java side as well.

This is part of a series of improvements to BQ. Part of the requirements is to include the launching step info along with the Dataflow job info.